### PR TITLE
[NEW] Add module <partner_prospect>

### PR DESCRIPTION
**Pull request by oihane**

### CODE REVIEW

**Comment by anajuaristi** [comment]
:-1: 

Tested in runbot... label is wrong. It's not "purchase order" but "sale order".

The idea is to filter in a click "real customers" that is... the ones who confirmed at less one sale order, nothing to see with purchase orders. 
Please change it.

Thank you

**Comment by oihane** [comment]
@anajuaristi Can you test again the module?

Now, it is taken into account the parent and childs of the partner, to calculate if it is a prospect or an actual customer.

Thanks

File: partner_prospect/README.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+* https://www.odoo.com/forum/help-1
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/sale-workflow/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/sale-workflow/issues/new?body=module:%20partner_handbill%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (pedrobaeza) [inline partner_prospect/README.rst]: Put `partner_prospect`

Comment (oihane) [inline partner_prospect/README.rst]: Oh, damn it...

**Comment by anajuaristi** [comment]
NICE!!! Tested in runbot.odoo-community.org

Now ok :+1: RTM

**Comment by petrus-v** [comment]
:+1:

Code review + tested on runbot!

File: partner_prospect/README.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Partner prospect
+================
```
Comment (max3903) [inline partner_prospect/README.rst:6]: Missing ===== above

File: partner_prospect/README.rst [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+With this module you can easily distinguise if a partner is currently customer
```
Comment (max3903) [inline partner_prospect/README.rst]: distinguish

File: partner_prospect/README.rst [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+or prospect. A prospect partner will be those to which you still haven't sold
+anything, considering sales as non cancelled nor draft sale orders to that partner.
+
+
+Usage
+=====
+
+To use this module, you need to:
+
+* go to ...
```
Comment (max3903) [inline partner_prospect/README.rst]: Where should we go ?

**Comment by oihane** [comment]
@max3903 required changes done in README.rst

**Comment by max3903** [comment]
:+1:

**Comment by rafaelbn** [comment]
Hi @oihane , even though the module is merged, please add a comment in usage about how it works. For example:
- You will find a check box in the partner names prospect, if checked it's prospect if not is client.
- **Contacts are clients or not depending of the Parent Company
- ...

Thanks!

**Comment by rafaelbn** [comment]
I have found a [BUG]

**Comment by citruscorp** [comment]
Estimado, con que etiqueta se podría subir una lista de clientes que son prospectos?

**Pull request merged**